cluster/gce/upgrade.sh

```diff
@@ -162,7 +190,9 @@ if [[ "${master_upgrade}" == "false" ]] && [[ "${node_upgrade}" == "false" ]]; t
   exit 1
 fi
 
-tars_from_version ${1}
```

Comment:
The call to "upload-server-tars" require the PROJECT env var to be set, which is only done in "detect-project". Having said that, I refactored the code a bit and pulled out the whole "prepare-upgrade" call out. I think this may be cleaner.